your-rents-data-jooq/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Do we need to have this in git?

Reply:
Yes, it allows to auto-download Maven if it's not already present.

Reply:
In fact we can remove the maven wrapper jar, but nor the MavenWrapperDownloader.java. I will remove it from all projects. https://github.com/takari/maven-wrapper#usage-without-binary-jar